lib/rest/swagger-doc.yaml

```diff
@@ -1,740 +1,931 @@
       type: object
+    Users:
       properties:
-        status:
-          type: string
-          description: new state of the Entity
-          example: Active
-  NotFound:
-    description: Entity not found.
-  ValidationError:
-    description: Supplied entity cannot be processed
-  EntityExists:
-    description: Conflict due to another resource already exists
+        users:
+          description: Array of Users
+          type: array
+          items:
+            '$ref': '#/components/schemas/User'
+        nextKey:
+          type: integer
+          description: Reserved for future needs, Always 0
+      type: object
+    Apps:
+      properties:
+        users:
+          description: Array of Apps
+          type: array
+          items:
+            '$ref': '#/components/schemas/App'
+        nextKey:
+          type: integer
+          description: Reserved for future needs, Always 0
+      type: object
+    Credentials:
+      properties:
+        credentials:
+          description: Array of Credentials
+          type: array
+          items:
+            '$ref': '#/components/schemas/Credential'
+        nextKey:
+          type: integer
+          description: Reserved for future needs, Always 0
+      type: object
+    Plugins:
+      properties:
+        plugins:
+          description: Array of Plugins
+          type: array
+          items:
+            '$ref': '#/components/schemas/PluginResponse'
+        nextKey:
+          type: integer
+          description: Reserved for future needs, Always 0
+      type: object
+    Plugin:
+      required:
+      - name
+      properties:
+        name:
+          type: string
+        description:
+          type: string
+        package:
+          type: string
+        version:
+          type: string
+        disabled:
+          type: boolean
+          default: false
+        state:
+          type: string
+          enum:
+            - awaiting-configuration
+            - configured
+          default: configured
+        options:
+          type: array
+          items:
+            '$ref': '#/components/schemas/PluginOption'
+      type: object
+    PluginOption:
+      properties:
+        name:
+          type: string
+        schema:
+          type: object
+        value:
+            oneOf:
+              - type: string
+              - type: boolean
+              - type: object
+              - type: integer
+              - type: number
+      type: object
+    PluginUpdateRequest:
+      allOf:
+        - '$ref': '#/components/schemas/Plugin'
+        - properties:
+            addPoliciesToGatewayConfig:
+              type: boolean
+              default: true
+    PluginResponse:
+      allOf:
+        - '$ref': '#/components/schemas/Plugin'
+        - example:
+            name: custom-plugin
+            description: An example plugin
+            package: express-gateway-plugin-custom
+            version: 1.1.0
```

Comment:
same here 1.2.0 ?

Reply:
This is the plugin version, not the configuration format version.